apicast/src/provider.lua

```diff
@@ -53,7 +54,8 @@ end
 
 function _M.configured(host)
   local hosts = _M.configuration:find(host)
-  return #hosts > 0
+
+  return next(hosts)
```

Comment:
Maybe we could return a `true` value, in case hosts is not empty (i.e. module is configured)? We're using it just to check if it's truthy, but it's kind of weird to receive a random object here, I would expect a `true` if configured, and `false` or `nil` otherwise...